src/test/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpointUnitTest.java

```diff
@@ -103,7 +103,6 @@
   private static final String CASE1_CASEGROUP_SAMPLE_UNIT_TYPE = "B";
   private static final String CASE_CI_ID = "40c7c047-4fb3-4abe-926e-bf19fa2c0a1e";
   private static final String CASE_PARTY_ID = "3b136c4b-7a14-4904-9e01-13364dd7b972";
-  private static final String CASE_ACTIONPLAN_ID_1 = "5381731e-e386-41a1-8462-26373744db81";
```

Comment:
this should be replaced with active enrolments and expectation would be to match with active enrolment